basic_pipeline/7_Depayloader.md

```diff
@@ -0,0 +1,140 @@
+Since we have packets put in order by the Ordering Buffer, we can assemble them into the original frames.
```

Comment:
Add sentence why it is called depayloader (it unpacks the payload from packets)

Reply:
done